Add missing options for chef-server-ctl backup and restore (#3577)
chef/chef-web-docs: themes/docs-new/layouts/shortcodes/ctl_chef_server_restore_options.md

:   Use to specify that the path to an empty directory to be used during
    the restore process. This directory must have enough disk space to
    expand all data in the backup archive.

`--pg-options`

> :   Specify additional options to pass to PostgreSQL during restore.

Reviewer: ```suggestion
:   Use to specify additional options to pass to PostgreSQL during backups. See the [PostgreSQL documentation](https://www.postgresql.org/docs/) for more information.
```

> :   Use to specify additional options to pass to PostgreSQL during restore. See the [PostgreSQL documentation](https://www.postgresql.org/docs/13/runtime-config.html) for more information.

Reviewer: ```suggestion
```suggestion
:   Use to specify and pass additional options PostgreSQL during backups. See the [PostgreSQL documentation](https://www.postgresql.org/docs/13/runtime-config.html) for more information.
```
```

`-t`, `--timeout`

> :   Set the maximum amount of time in seconds to wait for shell commands (default 600). This option should be set to greater than 600 for restores taking longer than 10 minutes.

Reviewer: ```suggestion
:   Set the maximum amount of time in seconds to wait for shell commands. Set to greater than 600 for backups that take longer than 10 minutes. Default: 600.
```
